README.md

```diff
@@ -159,46 +72,84 @@ The summary details will be accessible in `$RUNNER_TEMP/build-scan-data-maven/bu
       "requestedTasks": "install",
       "buildId": "1701252760831-89e3583d-3c3c-4a64-a958-e61aa791f4f4",
       "buildFailure": true,
+      "buildTimestamp": "1701252760800",
       "buildScanLink": "https://<DEVELOCITY_URL>/s/dxqnlj5hgybq4"
     }
   ]
 }
 ```
-By default, a summary will be added to the GitHub workflow calling the action (can be skipped with `skip-job-summary` is set to `true`):
+#### Capture unpublished Build Scan®
 
-![workflow](./doc/summary-workflow.png)
+The _capture strategy_ can be customized:
+- `ALWAYS`: default behavior, capture will be attempted on each Maven invocation
+- `ON_FAILURE`: capture will be attempted only on failed builds
+- `ON_DEMAND`: capture will be attempted if `CAPTURE_BUILD_SCAN=true` in the environment
+
+The _capture_ can be _enabled_/_disabled_ separately:
+- `capture-unpublished-build-scans`: to disable unpublished Build Scan® capture
+- `capture-build-scan-links`: to disable Build Scan® link capture
+
+The process is handled by a [Maven extension](https://maven.apache.org/guides/mini/guide-using-extensions.html) `maven-build-scan-capture-extension.jar` which is running during each Maven invocation.
```

Comment:
I wonder if this additional maven extension could cause build cache misses. Have we tested if it causes "goal implementation changed"?

Reply:
There is no "goal implementation changed" with the extension applied, the cache keys are consistent.
See [this comparison](https://ge.solutions-team.gradle.com/c/dmzfighnwjyro/n4smq2zqiug5k/goal-inputs) for instance

Reply:
Very nice!